index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/styles.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="header-item" id="logo-container">
+      <label>John Doe</label>
+    </div>
+      <button class="header-item" id="menu-button">
+        <img src="icons/Icon-Menu.png" alt="btn">
+      </button>
+  </header>
+
+  <section class="black-bg" id="headline">
+    <div class="bckgrnd right">
+      <img src="img/Illustration.png" alt="">
+    </div>
```

Comment:
- [ ] Please know that this illustration should be put as a background image in CSS using the  `background-image` property, kindly change that. 😁

Reply:
This image is a png, it doesn't cover all the background so I used it that way.